utils/AxiosRequest.js

```diff
@@ -22,9 +24,14 @@ export const fetchPredictions = async (userId, noOfRecommendations) => {
 	}
 };
 
-export const fetchAllArticles = async () => {
+export const fetchAllArticles = async (startIndex, noOfRecommendations) => {
+	const requestData = {
+		start_index: startIndex,
+		no_recommendations: noOfRecommendations,
+	};
+
   try {
-    const response = await axios.get(apiUrl + '/all', {
+    const response = await axios.post(apiUrl + '/all', requestData, {
```

Comment:
Forgor what this line does, why do we need apiUrl + /all?

Reply:
all articles/newest articles. kan godt ændre det til newest?